reporters/reporter-default/src/main/java/org/radargun/reporting/html/ReportDocument.java

```diff
@@ -41,6 +41,7 @@
 
    private int elementCounter = 0;
    private ArrayList<Future> chartTaskFutures = new ArrayList<>();
+   private ArrayList<String> generatedCharts = new ArrayList<>();
```

Comment:
This could use List interface (please change chartTaskFutures field & getter as well).